backend/app/routes.py

```diff
@@ -30,6 +48,8 @@ def closest_node(meters,longitude,latitude):
 # test URL /node/30357505
 @app.route('/node/<node_id>', methods=['GET'])
 def node(node_id):
+    """Returns information about a given node in the Here street network.
+    This uses the latest map version and may not recognize an older node_id."""

```

Comment:
For some functions, there are descriptions for the function arguments. For others like this, there aren't. Now of course, it is probably obvious what a node_id is, but maybe it's nice to describe that the id is the one associated with the here network.